src/nvim/main.c

```diff
@@ -793,6 +893,9 @@ static void command_line_scan(mparm_T *parmp)
           // "--version" give version message
           // "--noplugin[s]" skip plugins
           // "--cmd <cmd>" execute cmd before vimrc
+          // "--remote" open file on remote instance
+          // "--server" name of vim server to send to or name
+          // of server to become
```

Comment:
If we are going to introduce `--server` do we still need `--listen` ?

Reply:
`--server` was just from vim. but as the server names is quite different from vim anyway, I guess we can rename the option as well..